src/engine/Metavariable_regex.mli

```diff
@@ -0,0 +1,15 @@
+(* The first argument is the function
```

Comment:
Is this comment still valid?
The first parameter (not argument)  is an environment, not a function ...